rfc9002.md

```diff
@@ -367,7 +367,7 @@ smoothed_rtt.
 Endpoints SHOULD set the min_rtt to the newest RTT sample after persistent
 congestion is established. This avoids repeatedly declaring persistent
 congestion when the RTT increases. This also allows a connection to reset
-its estimate of min_rtt and smoothed_rtt after a disruptive network event;
+its estimate of min_rtt and smoothed_rtt after a disruptive network event;
```

Comment:
I like pink spaces better.

Reply:
My hex decoder tells me that this is non-breaking space.  How that got there is ... interesting.